tests/publiccloud/download_repos.pm

```diff
@@ -65,8 +65,12 @@ sub run {
         my $reject = "'robots.txt,*.ico,*.png,*.gif,*.css,*.js,*.htm*'";
         my $regex = "'s390x\\/|ppc64le\\/|kernel*debuginfo*.rpm|src\\/'";
         my ($incident, $type);
+        set_var("PUBLIC_CLOUD_EMBARGOED_UPDATES_DETECTED", 0);
```

Comment:
Do we need to use settings?Is it not a variable enough?

Reply:
I didn't get what you mean.

Reply:
why not to define a local variable
```
my $embargoed_updates_detected = 0;
```

And just use it?

Reply:
Because you cannot change the local variable without changing the code.

Reply:
on top of that - now this variable can me used in different `*.pm` files . If you make it local perl variable you can not read/write this state in other perl modules due to nature of how openQA run modules